services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/controller/DebugController.java

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
```

Comment:
Is this controller necessary for SubnetManager? Please clean up codes which are not needed.

Reply:
"Git pull upstream master" resolved the problem.